playground/css/vite.config.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
   },
   resolve: {
     alias: {
-      '@': __dirname,
+      '=': __dirname,
```

Comment:
Replaced this to test less's variable. (less's variable starts with `@`)

related: https://github.com/vitejs/vite/pull/10741/files#diff-2cfbd4f4d8c32727cd8e1a561cffbde0b384a3ce0789340440e144f9d64c10f6R1608